src/here-xyz.ts

```diff
@@ -1900,43 +1912,58 @@ async function configXyzSpace(id: string, options: any) {
     }
 
     if(options.schema) {
-        if (spacedef.processors) {
-            let i = spacedef.processors.length;
-            while (i--) {
-                let processor = spacedef.processors[i];
-                if (processor.id === 'schema-validator') {
-                    spacedef.processors.splice(i, 1);
+        if(options.schema == true && options.delete != true) {
+            if(spacedef.processors) {
+                let i = spacedef.processors.length;
+                while (i--) {
+                    let processor = spacedef.processors[i];
+                    if (processor.id === 'schema-validator') {
```

Comment:
we should give proper message when no schema definition is found

Reply:
thank you for raising this, IMHO at this point the schema is expected to be found since the URL and schema def placement is done by hub during the configuration itself, and if some technical failure occurs ( like network related etc ), we handle it in the common error handling method already.  If you are satisfied with the answer please mark the conversation resolved although any feedback is always welcome.

Reply:
I missed the console log of "no schema found". My intention of comment was that message only.